test/apps/errors/src/routes/_layout.svelte

```diff
@@ -3,6 +3,14 @@
 	const { page } = stores();
 </script>
 
+<script context="module">
+  export function preload({ params, query }) {
```

Comment:
spaces here should be tabs

Reply:
@ehrencrona it looks like this one still needs to be updated

Reply:
sorry, missed that comment